docs/cody/enterprise/completions-configuration.mdx

```diff
@@ -0,0 +1,186 @@
+# Completions Configuration
+
+<p className="subtitle">Learn how to configure Cody via `completions` on a Sourcegraph Enterprise instance.</p>
+
+<Callout type="info">Configuring Cody via `completions` is legacy but it's still supported. We recommend using the new [`modelConfiguration`](/cody/enterprise/model-configuration) for flexible LLM model selection.</Callout>
+
+[Cody Enterprise](https://sourcegraph.com/enterprise) supports many models and model providers. You can configure Cody Enterprise to access models via Sourcegraph Cody Gateway or directly using your own model provider account or infrastructure. Let's look at these options in more detail.
+
+## Using Sourcegraph Cody Gateway
+
+This is the recommended way to configure Cody Enterprise. It supports all the latest models from Anthropic, OpenAI, Mistral, and more without requiring a separate account or incurring separate charges. You can learn more about these in our [supported models](/cody/capabilities/supported-models) docs.
+
+## Using your organization's account with a model provider
+
+### Example: Your organization's Anthropic account
+
+First, [create your own key with Anthropic](https://console.anthropic.com/account/keys). Once you have the key, go to **Site admin > Site configuration** (`/site-admin/configuration`) on your instance and set:
+
+```json
+{
+  // [...]
+  "cody.enabled": true,
+  "completions": {
+    "provider": "anthropic",
+    "chatModel": "claude-2.0", // Or any other model you would like to use
+    "fastChatModel": "claude-instant-1.2", // Or any other model you would like to use
+    "completionModel": "claude-instant-1.2", // Or any other model you would like to use
```

Comment:
Nit: This is more straightforward, but it'd be better if the admin could be sure that this is specifically for _code autocompletions_, not e.g. chat completions. (We call "chat completions" just "completions" in our code, not sure about definitions in customers' heads and even other places of our own docs.)

Reply:
Ah okay. I read all the green lines of the PR from top to bottom, tried to empathize with the admin reading it, and left my comments. Then my suggestion is probably for @MaedahBatool.